.github/workflows/test-js-packages.yml

```diff
@@ -559,7 +559,7 @@ jobs:
           - project: { repo: AztecProtocol/aztec-packages, path: ./noir-projects/noir-protocol-circuits/crates/rollup-lib }
           - project: { repo: AztecProtocol/aztec-packages, path: ./noir-projects/noir-protocol-circuits/crates/types }
 
-    name: Check external repo - ${{ matrix.project.repo }}
+    name: Check external repo - ${{ matrix.project.repo }}/${{ matrix.project.path }}
```

Comment:
nit: If we add this, let's remove `./` from the path above as the output of this line will then be `AztecProtocol/aztec-packages/./noir-projects/noir-protocol-circuits/crates/types` which is a bit uglier than if it were not there

Reply:
If you remove it, it also appears [here](https://github.com/noir-lang/noir/blob/78f5e87c3fee3a51a79b1c7f6e41572e8e35029f/.github/workflows/test-js-packages.yml#L537)

Reply:
Will it work if I change "./" to "" there?

Maybe I can leave these changes to a separate PR? (and undo the one I did)

Reply:
It should work, it's only used as a [suffix](https://github.com/noir-lang/noir/blob/78f5e87c3fee3a51a79b1c7f6e41572e8e35029f/.github/workflows/test-js-packages.yml#L593) to `/test-repo/`

But sure, it's unrelated, I thought the PR may have tightened some reporting, but it fails everywhere.

Reply:
Done!